index.html

```diff
@@ -48,6 +48,7 @@
     <section>
       <h1>Index of my presentations</h1>
       <ul>
+        <li><a target="_blank" href="./web-platform-when-why/index.html">The Right Choice: When is the Web is the right answer —and why?</a></li>
         <li><a target="_blank" href="./syntactic-macros-python/index.html">Syntactis macros in Python!</a></li>
```

Comment:
Typo, 'syntactis' instead of 'syntactic'. Just in case the 's' wasn't a typo though, adjectives don't have number!